test/compilable/test23047.d

```diff
@@ -0,0 +1,15 @@
+/* REQUIRED_ARGS: -O
+ */
+
+// https://issues.dlang.org/show_bug.cgi?id=23047
+
+import core.simd;
+
+version (D_SIMD):
+
+long2 _mm_srl_epi64 ()
+{
+    long2 r = void;
```

Comment:
You can avoid importing `core.simd` and use `__vector` directly instead. It will improve compile times on the test suite if you do `-defaultlib= -c -O`.